tests/config/test_data.config

```diff
@@ -1,5 +1,6 @@
 // Base directory for test data
 def test_data_dir = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data"
+def test_peps_data_dir = "https://raw.githubusercontent.com/pepkit/example_peps/master"
```

Comment:
Why are you using these rather than the nf-core/test-datasets you added: https://github.com/nf-core/test-datasets/tree/modules/data/delete_me/pep

Also you shouldn't have to update this file at all technically, as we've put evertyhing in the `delete_me` folder.